code/game/objects/items.dm

```diff
@@ -632,3 +639,31 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 		M.dirty++
 
 /obj/item/proc/on_mob_death(mob/living/L, gibbed)
+
+/obj/item/proc/friendly_force()
+	if(force)
+		if(force <= 3)
+			f_force = "very low"
+		else if(force <= 5)
+			f_force = "low"
+		else if(force <= 7)
+			f_force = "medium"
+		else if(force <= 10)
+			f_force = "<font color=green>high</font>"
+		else if(force <= 15)
+			f_force = "<font color=red>robust</combat>"
+		else if(force <= 24)
+			f_force = "<font color=orange>very robust</font>"
+		else if(force >= 25)
+			f_force = "<font color=white>Exceptionally Robust</font>"
+
+
+/obj/item/MouseEntered(location,control,params)
+	if(in_inventory && datum/preferences/enable_tips)
+		friendly_force()

```

Comment:
*ahem.

you didn't do the caching and comparison I suggested.
I know it's relatively cheap but there's no point adding proc call overhead where its avoidable.